src/en/controllers-config.md

```diff
@@ -0,0 +1,69 @@
+Title: General configuration options
+TODO: Check accuracy of key table
+      Make the table more space-efficient. Damn it's bulbous.
+
+
+# Configuring controllers
+
+A Juju controller is the management node of a Juju cloud environment. It houses
+the database and keeps track of all the models in that environment.
+
+Controller configuration consists of a collection of keys and their respective
+values. An explanation of how to both view and set these key:value pairs is
+provided below. Notable examples are provided at the end.
+
+
+## Getting and setting values
+
+You can display the current model settings by running the command:
+
+```bash
+juju controller-config
+```
+
+This will include all the currently set key values - whether they were set
+by you, inherited as a default value or dynamically set by Juju. 
+
+A key's value may be set for the current model using the same command:
+
+```bash
+juju controller-config auditing-enabled=true
```

Comment:
Probably worth noting that many of the settings are read only once the controller has been bootstrapped.